src/sagas/systemIntakeSaga.ts

```diff
@@ -20,6 +21,24 @@ export function* saveSystemIntake(payload: SaveSystemIntakeAction) {
   }
 }
 
+function requestSystemIntake(accessToken: string, id: string) {
```

Comment:
This can get changed to getSystemIntake when we rename everything (or maybe fetchSystemIntake)